notesserver/settings/yaml_config.py

```diff
@@ -43,3 +42,6 @@
 if ES_DISABLED:
     HAYSTACK_CONNECTIONS = {}
     INSTALLED_APPS.remove('haystack')
+
+LOGGING = build_logging_config()
```

Comment:
Ah, used here.  Can we move the import here instead?

Reply:
Yes that seems smart. Updated.